active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
+- Raise awareness of the Django Web Framework and its community with the broader world
+
+Delegated responsibilities:
+
+- Contacting, developing and implementing financial sponsorships with donor sources
+- Clarifying and elaborating on the various creative strategies that donor sources of different sizes and types can use to support Django
 
-Delegated responsibilities TBD.
 
 ## Initial membership
 
 - Chair: TBD
 - Co-Chair: TBD
 - Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
-- Other members: TBD
+- Other members:
+  - Jacob Kaplan-Moss
+  - Priya Pahwa
+  - Thibaud Colas
+  - Tom Carrick
+  - Sarah Abderemane
+  - Stephanie Goulet
+
 
 ## Future membership
 
 Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
 
-This committee is currently forming and seeking volinteers, including chair and co-chair. 
+This committee is currently forming and seeking volunteers, including chair and co-chair. 
+
+### Navigating Conflicts of Interest
+Each new member must disclose any potential conflicts of interest. These disclosures will be made transparent by means of ____. 
+
+Any person with a conflict of interest will need to recuse themselves from any votes or other decision-making related to their conflict.
+
+No more than X% of the Fundraising Working Group may contain individuals associated with the same organization
+
 
 ## Budget
 
 No allocated funds.
 
 ## Comms
 
-TBD
+- Collaborative documents
+- Private channel in DSF slack
+- At least monthly synchronous meetings
+
 
 ## Reporting
 
-TBD
+We'll email a written report to the board every quarter.
```

Comment:
I’d expect more regular updates, and most likely a shared dashboard with active fundraising leads or similar CRM-type data. This is quite critical information for the board to be aware of and have a chance to discuss as part of every meeting.

Reply:
You could copy some language from what I used in #15 -- https://github.com/django/dsf-working-groups/blob/bde7b66cd4204e22a1b7418aa91fb7c5da2c0cb3/active/fellowship.md?plain=1#L60-L68